package/scripts/presto_coordinator.py

```diff
@@ -62,6 +62,8 @@ def configure(self, env):
                 # the user visible equivalent to node-scheduler.include-coordinator
                 if key == 'pseudo.distributed.enabled':
                     continue
+                if key in memory_configs:
```

Comment:
Short answer: no.

Unfortunately, you can't pick dynamically between MB and GB. The Ambari stylesheet (for lack of a better word), makes you pick a unit and you can't change it without re-installing the Presto extension. I think keeping GB hardcoded is fine. For most deployments a minimum of 1GB for most memory configs is fine.